packages/zoe/src/contractFacet/vatRoot.js

```diff
@@ -42,10 +41,10 @@ export async function buildRootObject(powers, vatParameters, baggage) {
     baggage.init('zoeService', zoeService);
     baggage.init('invitationIssuer', invitationIssuer);
   } else {
-    assert(!zoeService);
+    assert(!zoeService, 'zoeService is required');
```

Comment:
the message doesn't match the assertion. Isn't this ensuring that `zoeService` was _not_ provided by `vatParamters`?